commons/pdf-commons/src/main/java/se/idsec/signservice/security/sign/pdf/document/VisibleSignatureImage.java

```diff
@@ -240,7 +240,7 @@ private String getPersonalizedSvgImage(final String svg, final Date signingTime)
     }
 
     if (this.includeDate) {
-      personalizedJson = personalizedJson.replaceAll("##SIGNTIME##", VisibleSignatureImage.BASIC_DATE_FORMAT.format(signingTime));
+      personalizedJson = personalizedJson.replaceAll("##SIGNTIME##", dateFormat.format(signingTime));
```

Comment:
The ##SIGNTIME## does not correspond to the symbolic name set in the API's PdfSignatureImageTemplate. There we have: 
```
public static final String SIGNING_TIME_FIELD_NAME = "signingTime";
```

We need to change this. Either here or there.